packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -4030,9 +3997,6 @@ describe('ReactSuspenseWithNoopRenderer', () => {
         // Neither is resolved yet.
         'Pending...',
         '',
-        'Pending...',
-        'Suspend! [a]',
```

Comment:
How was [a] suspending again before and why isn't it now?

Reply:
A used to be categorized as `TransitionShortPriority` and B  was `TransitionLongPriority`. Now they're both just `TransitionPriority` and so get batched together. This will change again once we implement parallel transitions.

Reply:
I see, would it help to update the comment here to say:

```
// Neither is resolved yet, but they're batched so we only see the B suspend flushed.
```